letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
Yes, that's why I point to check /web/database/selector and if there are two or more databases in combo list, then --load is needed.